bundles/AdminBundle/Resources/public/js/pimcore/notification/plugin.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+pimcore.registerNS("pimcore.notification.plugin");
+
+pimcore.notification.plugin = Class.create({
+
+    getClassName: function () {
+        return "pimcore.notification.plugin";
+    },
+
+    initialize: function () {
+        var self = this;
+
+        pimcore.plugin.broker.registerPlugin(this);
+
+        var element = '<li style="display: none" id="pimcore_notification" data-menu-tooltip="' + t("Notifications") + '" class="pimcore_menu_needs_children">'
+            + '<svg id="gicsgdfk" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48">'
+            + '<path style="fill:#78909C;" d="M42.3,12.8c1.1,0.7,1.7,2,1.7,3.3V37c0,2.2-1.8,4-4,4H8c-2.2,0-4-1.8-4-4V16.1c0-1.3,0.6-2.5,1.7-3.3"/>'
+            + '<path style="fill:#CFD8DC;" d="M40,41H8c-2.2,0-4-1.8-4-4V17l20,13l20-13v20C44,39.2,42.2,41,40,41z"/>'
+            + '</svg>'
+            + '<span id="notification_value" style="display:none;"></span>'
+            + '</li>';
+
+        this.navEl = Ext.get("pimcore_menu_search").insertSibling(element, "after");
+        this.menu = new Ext.menu.Menu({
```

Comment:
hmm, what do you think about this idea: 
not put this entry as a main menu entry, but add it to the icons at the bottom like debug mode, dev mode, missing settings etc.?

Reply:
Yes, good idea. I moved at the bottom as the first icon above debug.

Reply:
just checked that, works!